libr/core/cmd_print.c

```diff
@@ -4486,6 +4486,111 @@ static void r_core_disasm_table(RCore * core, int l, const char *input) {
 	r_table_free (t);
 }
 
+static void cmd_pxr(RCore *core, int len, int mode, int wordsize, const char *arg) {
+	PJ *pj = NULL;
+	RTable *t = NULL;
+	if (mode == ',') {
+		t = r_table_new ();
+		RTableColumnType *n = r_table_type ("number");
+		RTableColumnType *s = r_table_type ("string");
+		r_table_add_column (t, n, "addr", 0);
+		r_table_add_column (t, n, "value", 0);
+		r_table_add_column (t, s, "refs", 0);
+	}
+	if (mode == 'j') {
+		pj = pj_new ();
+		if (!pj) {
+			return;
+		}
+	}
+	if (mode == 'j' || mode == ',' || mode == '*' || mode == 'q') {
+		size_t i;
+		const int base = core->anal->bits;
+		if (pj) {
+			pj_a (pj);
+		}
+		const ut8 *buf = core->block;
+		int withref = 0;
+		for (i = 0; i + wordsize < core->blocksize; i += wordsize) {
+			ut64 addr = core->offset + i;
+			ut64 val = buf[i];
+			if (i + wordsize >= len) {
+				break;
+			}
+			if (base == 32) {
+				val = r_read_le32 (buf + i);
+				val &= UT32_MAX;
+			} else {
+				val = r_read_le64 (buf + i);
+			}
```

Comment:
base can be only 32 or 64 bits? and only little endian?

Reply:
for the 99% of real use cases yes. but i have extended it in a commit